### ISSUE TITLE
flutter_cast_video:compileDebugKotlin

### ISSUE THREAD

**Issue by priyank-dt** [whowillcare/flutter_cast_video#16]
Hii @whowillcare  
I get below while compiling the code, Can you please give me solution for this ?

Execution failed for task ':flutter_cast_video:compileDebugKotlin'.
> Inconsistent JVM-target compatibility detected for tasks 'compileDebugJavaWithJavac' (1.8) and 'compileDebugKotlin' (17).

**Comment by iameerhamza** [comment]
@priyank-dt did you solved the issue?

**Comment by mrmostafaei** [comment]
@iameerhamza did you solved the issue?

**Comment by iameerhamza** [comment]
@mrmostafaei yes, my current configuration is working fine , the problem was just with gradle versions and other groovy settings

**Comment by mrmostafaei** [comment]
@iameerhamza can you share your configuration? Which version of gradle works correctly?

**Comment by priyank-dt** [comment]
> @priyank-dt did you solved the issue?

No, I'm still experiencing the issue. Do you have any solutions you could share if you've already resolved it?

**Comment by iameerhamza** [comment]
@mrmostafaei try upgrading your gradle version to the latest version im currently using 
`distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-all.zip`

and also open your android root folder in android studio and update the settings and recommendation from there
other than that

this is my android/build.gradle file

`allprojects {
    repositories {
        google()
        mavenCentral()  // Maven Central repository
    }
}

afterEvaluate { project ->
    if (project.hasProperty("kotlin")) {
        project.tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
            kotlinOptions {
                jvmTarget = "1.8"
            }
        }
    }
}

rootProject.buildDir = '../build'

subprojects {
    afterEvaluate { project ->
        if (project.hasProperty('android')) {
            project.android {
                if (namespace == null) {
                    namespace project.group
                }
            }
        }
    }
}
subprojects {
    project.buildDir = "${rootProject.buildDir}/${project.name}"
    project.evaluationDependsOn(':app')
}

subprojects {
    project.configurations.configureEach {
        resolutionStrategy.eachDependency { details ->
            if (details.requested.group == 'com.android.support'
                    && !details.requested.name.contains('multidex') ) {
                details.useVersion "27.1.1"
            }
        }
    }
}
tasks.register("clean", Delete) {
    delete rootProject.buildDir
}
`

this is my settings.gradle file

pluginManagement {
    def flutterSdkPath = {
        def properties = new Properties()
        file("local.properties").withInputStream { properties.load(it) }
        def flutterSdkPath = properties.getProperty("flutter.sdk")
        assert flutterSdkPath != null, "flutter.sdk not set in local.properties"
        return flutterSdkPath
    }
    settings.ext.flutterSdkPath = flutterSdkPath()

    includeBuild("${settings.ext.flutterSdkPath}/packages/flutter_tools/gradle")

    repositories {
        google()
        mavenCentral()
        gradlePluginPortal()
    }
}

plugins {
    id "dev.flutter.flutter-plugin-loader" version "1.0.0"
    id "com.android.application" version '8.7.1' apply false
    id "org.jetbrains.kotlin.android" version "1.9.24" apply false
    id "com.google.gms.google-services" version "4.4.2" apply false
}

include ":app"

**Comment by markst** [comment]
Network graph reveals there has been contributions: https://github.com/whowillcare/flutter_cast_video/network

However I'm unable to open PR as I'm not a collaborator
